frontend/scss/styles.scss

```diff
@@ -153,3 +153,12 @@ footer {
         background-color: $color-red !important;
     }
 }
+
+.btn-secondary-blue {
+    @extend .btn-secondary;
+    border: .08em solid $color-blue-light;
+    &:hover, &:focus {
+        color: white !important;
```

Comment:
When using @extend, it's good practice to only add styles that are different. Since `color: white !important` is already defined in the parent `.btn-secondary`, you can remove this line.